### PR TITLE
[#1205] Provide more information on the landing page

### LINKED ISSUE #1205: Provide help page in dashboard
The current workflow of RepoSense is that users can setup a RepoSense dashboard and share the link to his/her peers or students. 

However, the people that are shared to may not understand how to interpret the report (such as what does the ramp size and ramp position represent etc). They are also highly unlikely to go to our User Guide to find out how to interpret the report. 

Perhaps we can provide sort of a help page (in terms of a modal) that provides all information to interpret the report? 

_p.s. asking for this because there were several occasions where i shared the dashboard to my peers but they can't understand what the ramp represents._

### PR DESCRIPTION
**Pull request by dcshzj**
Part of #1205.

```
The report is not easy to understand for a first-time user, as
there is a lack of a quick introduction on the landing page.

Let's provide some basic information on some of the useful
functionality and provide a direct link to the relevant user guide.
```

### CODE REVIEW

**Comment by Tejas2805** [comment]
The issue kind of focuses on the fact that users will not want to go through the UG to understand how to read the report. But seems like a good start.

**Comment by dcshzj** [comment]
Yep @Tejas2805, I identified 2 issues as both a first-time user and contributor which I tried to solve with this PR:

1. What are the features of RepoSense that I can take advantage of when I first reach this page?

I take for instance the link from CS2103 websites where it is labeled "iP Code Dashboard" which directs users to RepoSense, I'm not sure what to expect when I first arrive. Instead, I try to highlight the two main features of RepoSense that first-time users can take advantage of and from there, it introduces what RepoSense is about.

2. Where can I go to read more information on how to use RepoSense?

The current user guide is linked at the bottom of the page after all the repositories. Especially for a large student population, the user will have to "find" the correct link. Here, I direct them to the correct link in the user guide right when the user first enters the website.

**Comment by fzdy1914** [comment]
Sorry for the confusion in naming, but can you request the review from reposense/stage-1-active-reviewer instead of stage1 reviewer?

File: frontend/src/static/js/main.js [hunk 1]
```diff
@@ -209,6 +209,14 @@ window.app = new window.Vue({
       return `${window.HOME_PAGE_URL}/RepoSense/ug/index.html`;
     },
 
+    getReportUserGuideLink() {
```
Comment (Tejas2805) [inline frontend/src/static/js/main.js:212]: Might be better to name it `getUsingReportLink()` I think.

File: frontend/cypress/tests/codeView/codeView.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ describe('code view', () => {
         .should('be.visible');
 
     cy.get('#tab-empty > .title').then(($ele) => {
-      const expected = 'To view the code attributed to a specific author, click the    icon next to that author\'s name.To hide the code view, click the    icon on the left.';
+      const expected = 'Welcome to RepoSense!The charts on the left shows the contribution activities, grouped by repository and author.To view the code attributed to a specific author, click the    icon next to that author\'s name.To view the breakdown of commits made by a specific author, click the    icon next to that author\'s name.To hide the code view and show only the activity charts, click the    icon on the left.Please see the  user guide  for more information on using the report.';
```
Comment (Tejas2805) [inline frontend/cypress/tests/codeView/codeView.js:7]:
```suggestion
      const expected = 'Welcome to RepoSense!The charts on the left show the contribution activities, grouped by repository and author. To view the code attributed to a specific author, click the    icon next to that author\'s name. To view the breakdown of commits made by a specific author, click the    icon next to that author\'s name. To hide the code view and show only the activity charts, click the    icon on the left. Please see the  user guide  for more information on using the report.';
```

File: frontend/src/index.pug [hunk 3]
```diff
@@ -80,13 +80,27 @@ html
...
+                    br
+                    | To hide the code view and show only the activity charts, click the &nbsp;
+                    font-awesome-icon(icon="caret-right")
+                    | &nbsp; icon on the left.
+                  p
+                    | Please see the &nbsp;
+                    a(
+                      v-bind:href="getReportUserGuideLink()", target="_blank"
+                    )
+                      strong user guide
```
Comment (Tejas2805) [inline frontend/src/index.pug:102]:
```suggestion
                      strong User Guide
```

File: frontend/src/index.pug [hunk 4]
```diff
@@ -80,13 +80,27 @@ html
...
+                    | &nbsp; for more information on using the report.
```
Comment (Tejas2805) [inline frontend/src/index.pug:103]:
```suggestion
                    | &nbsp; to get a better understanding of how to interpret the report.
```

**Review by Tejas2805 — CHANGES_REQUESTED** [review]
Have left a few comments.

**Comment by damithc** [comment]
> @damithc Since it affects the UI though to a very small extent can get your review on this?

image

Some comments/queries:
* This is not the RepoSense website. It's just a report generated by RepoSense. `Welcome to this RepoSense report`?
* `shows` -> `show`? Not sure.
* Are you using images for icons? If yes, shouldn't you use fontawesome instead?
* Last point: `... on the left` -> `... on the divider`?
* `Please see`: remove `Please`

**Comment by dcshzj** [comment]
@damithc I incorporated some of the proposed changed. As for the icons, they are indeed using font awesome icons.

**Comment by damithc** [comment]
> @damithc I incorporated some of the proposed changed. As for the icons, they are indeed using font awesome icons.

OK for me :-)

**Review by jamessspanggg — APPROVED** [review]
LGTM 👍🏻

**Review by Tejas2805 — APPROVED** [review]
LGTM!

**Review by fzdy1914 — APPROVED** [review]

**Pull request merged**